plugins/woocommerce/changelog/wooplug-3844-add-conditional-options-for-displaying-weight-dimensions-and

```diff
@@ -0,0 +1,4 @@
+Significance: patch
+Type: add
+
+Add settings to control the visibility of weight/dimensions/attribute.
```

Comment:
Worth adding some context. Changelog is WooCommerce-wide so it might be hard to get where was it added 😄 

```suggestion
Product Specifications: Add settings to control the visibility of weight/dimensions/attribute.
```

Reply:
Right, we need more context. Thank you!